ts/components/session/SessionPasswordPrompt.tsx

```diff
@@ -137,15 +142,15 @@ export class SessionPasswordPrompt extends React.PureComponent<{}, State> {
       });
     }
 
-    // Prevent pating into input
+    // Prevent pasting into input
     return false;
   }
 
   public async onLogin(passPhrase: string) {
-    const trimmed = passPhrase ? passPhrase.trim() : passPhrase;
+    const passPhraseTrimmed = passPhrase.trim();
 
     try {
-      await window.onLogin(trimmed);
+      await window.onLogin(passPhraseTrimmed);
     } catch (error) {
       // Increment the error counter and show the button if necessary
       this.setState({
```

Comment:
Not an issue with this PR but is that a good idea to trim the passphrase? I mean if a user uses a password with by any luck a space at the end? It's ok for me but we should double check every time we save the password we trim it also.
Otherwise the user might be able to save a password with a space at the end but will never be able to login with it.

Reply:
Makes sense. I'll address this in a new PR